modules/aioseop_opengraph.php

```diff
@@ -203,6 +203,27 @@ function __construct() {
 			$this->init();
 		}
 
+		/**
+		 * Handle specific meta tags.
+		 *
+		 * @param mixed $value The value of the meta tag.

```

Comment:
Is this a string, or an array...or both? Using 'mixed' doesn't explain the type much, but 'string|array' would be more detailed; assuming it's 'mixed' because of an array.